Streetcode/Streetcode.BLL/MediatR/Authentication/RefreshToken/RefreshTokenHandler.cs

```diff
@@ -18,28 +18,28 @@ public RefreshTokenHandler(ITokenService tokenService, ILoggerService logger)
             _logger = logger;
         }
 
-        public async Task<Result<RefreshTokenResponceDTO>> Handle(RefreshTokenQuery request, CancellationToken cancellationToken)
+        public Task<Result<RefreshTokenResponceDTO>> Handle(RefreshTokenQuery request, CancellationToken cancellationToken)
         {
-            JwtSecurityToken? token = null;
+            JwtSecurityToken token = null;
```

Comment:
Don't you get warning here?
![image](https://github.com/user-attachments/assets/afa5c94a-ed50-4140-b080-7d750946c18d)